components/NextSteps.tsx

```diff
@@ -1,11 +1,13 @@
 import { Shimmer } from './Shimmer'
```

Comment:
ooo another sticky merge: I think you take priority for on the nextStep typing, but you should pull in the structure as redone on main (sorry!)